Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -1288,7 +1294,7 @@ const DEFAULT_VALUES = {
   defaultToWebgl2: true, // attempt webgl2 on by default
   activeFramebuffer: null,
   xrSession: null,
-  xrSessionIsAR: false,
+  xrSessionType: null,
```

Comment:
This should be updated in index.d.ts. Alternatively, get rid of `xrSessionIsAR` from the initial values interface, if it's not meaningful for the user to set it on construction.

Reply:
Removed